src/main.c

```diff
@@ -52,6 +52,7 @@ static const struct option prog_opts[] = {
 	{ "certpath", required_argument, 0, 'C' },
 	{ "time", no_argument, 0, 't' },
 	{ "no-boot-update", no_argument, 0, 'b' },
+	{ "migrate", no_argument, 0, 'b' },
```

Comment:
The subject of this commit is "Add better error handling throughout and cleanup", so I did not expect to see a new command line option added here. I would prefer to see it added in a separate commit so that I can make better sense of the changes.